doc/specifications/identities/Remote-identities-creation-diagram.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+actor "Authenticated user" as user
+participant "Client" as client
+database "Backend" as backend
+user -->o client: click\n« add remote account »
+client -> backend: ""GET /providers""
+backend -> client: providers list\nwith additional data\nto pre-fill form
+hnote left client: UI : « choose a provider »
+note over user
+User
+1. select a provider
+2. fill remote address, name, etc.
+3. fill connection settings (if required)
+         (for gmail and twitter,
+         server settings are pre-filled)
+4. if applicable :
+        select auth method : password or Oauth
+5. if applicable : fill credentials
+        (for ex. password auth)
+end note
+user -->o client: save form
+client -> backend: ""POST /identities/remotes""
```

Comment:
In case of failure of authorizing the user (closing popup for example), this mechanism should be the same for PATCH and might regenerate the `Oauth parameters` in response to the client